include/mbedtls/config.h

```diff
@@ -2910,6 +2910,7 @@
 //#define MBEDTLS_CTR_DRBG_MAX_INPUT                256 /**< Maximum number of additional input bytes */
 //#define MBEDTLS_CTR_DRBG_MAX_REQUEST             1024 /**< Maximum number of requested bytes per call */
 //#define MBEDTLS_CTR_DRBG_MAX_SEED_INPUT           384 /**< Maximum size of (re)seed buffer */
+//#define MBEDTLS_CTR_DRBG_USE_128_BIT_KEY              /** use 128 bit key for CTR-DRBG - redcuces security */
```

Comment:
The comment needs to explain in what way this reduces security. If the explanation is long, this can be a reference to another file such as `ctr_drbg.h`.